src/Containers/Profile/profileContainer.jsx

```diff
@@ -0,0 +1,236 @@
+import axios from 'axios';
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import Input from '../../Components/Common/input/input';
+import { viewProfile, editProfile } from '../../actions/profileActions';
+import './Profile.scss';
+import NavBar from '../../Components/NavBar/NavBar';
+import SideBar from '../../Components/SideBar/SideBar';
+import picture from '../../assets/img/blank_profile_pic.png';
+import initialState from '../../store/initialState';
+
+const CLOUDINARY_URL = 'https://api.cloudinary.com/v1_1/ah-wakanda/image/upload';
+const CLOUDINARY_UPLOAD_PRESET = 'j9eazq6w';
+
+/**
+ *
+ * @param {event} e
+ * @returns {method} render
+ */
+export class ProfileContainer extends Component {
+  state = {
+    bio: '',
+  };
+
+  fileInput = React.createRef();
+
+  /**
+   *
+   * @returns {method} form
+   */
+  componentWillMount() {
+    const { onViewProfile } = this.props;
+    const { username } = initialState.currentUser.isAuth;
```

Comment:
isAuth in currentUser is just a boolean variable, can you take the username from  currentUser.user instead ?

Reply:
Thank you for the feedback, it happened when I was troubleshooting and I forgot. But now it is changed it.